portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -43,11 +43,13 @@
   private static final String COMMENT_AMOUNT = "amount";
   private static final String COMMENT_TEXT = "text";
   private static final String COMMENT_TIMESTAMP = "timestamp";
+  private static final String COMMENT_NAME = "name";
+  
+  private static final String ANONYMOUS = "anonymous";
```

Comment:
nit: consider renaming this to ANONYMOUS_AUTHOR

Reply:
done!